LingqAnkiSync/AnkiHandler.py

```diff
@@ -42,18 +57,39 @@ def CreateNoteType(name: str, fields: List):
         mw.col.models.addField(model, mw.col.models.newField(field))
 
     template = mw.col.models.newTemplate("lingqAnkiSync")
-    template['qfmt'] = "{{Front}}<br>{{Sentence}}"
-    template['afmt'] = "{{FrontSide}}<hr id=answer>{{Back}}"
+    resourceFolder = os.path.dirname(__file__) + "/resources"
```

Comment:
This will likely break on windows. 
```suggestion
    resourceFolder = os.path.join(os.path.dirname(__file__), "resources")
```

Reply:
Oh good catch 👍